### ISSUE TITLE
`HEA` vs `HEB`  vs `IPE` ? 🤷‍♂️ 

### ISSUE THREAD

**Issue by nelsonic** [dwyl/learn-civil-engineering#17]
_Really_ wish I either studied Structural Engineering at University or had a "buddy" I could call on for this ... 🙄 
All the people I know that studied Civil Engineering are doing a completely different job ... 🤦‍♂️

Trying to finalise my decision of which type of mild steel beam to use ... 
So far I've been using `IPE` (i-Beam) in the design because they offer good price-strength (cost-benefit).
But I'm leaning toward `HEA` because it's _much_ stronger for the equivalent dimensions. 

Our _local_ metal merchants (we have bought a bunch of our steel from) have the following "pocket guide":
https://peixotorodrigues.pt/wp-content/uploads/2022/12/Guia-de-bolso-digital-Peixoto-Rodrigues.pdf
PDF: [Guia-de-bolso-digital-Peixoto-Rodrigues.pdf](https://github.com/dwyl/learn-civil-engineering/files/11584971/Guia-de-bolso-digital-Peixoto-Rodrigues.pdf)

<img width="1416" alt="IPE-safety-load-table" src="https://github.com/dwyl/learn-civil-engineering/assets/194400/5a8bd3ed-2b47-4759-901c-0e40b9b55717">

<img width="1412" alt="HEA-safety-load-table" src="https://github.com/dwyl/learn-civil-engineering/assets/194400/cfbeffc9-f543-4ae3-bced-0e180f0fccf6">

<img width="1411" alt="HEB-safety-load-table" src="https://github.com/dwyl/learn-civil-engineering/assets/194400/5ade2e09-5457-440d-bad5-1e23e7648d81">

I've highlighted the **`120`** line on each of the page screenshots. 
The reason I'm highlighting up to the `5mt` column in the table is because that is the max length in our project.

**Comment by iteles** [comment]
Tio Didi? We were thinking of inviting them to lunch next week anyway

**Comment by nelsonic** [comment]
Yes, `Tio Didi` has offered to take a look a the CAD drawing. I just don't have it _finished_ yet. ⏳ 
Part of my goal with using `FreeCAD` is to have a _parametric_ model that we can change thing _relatively_ easily. 🤞 
Going to keep going on the model. 👷‍♂️

**Comment by nelsonic** [comment]
An older but still relevant forum post: https://forumdacasa.com/discussion/16211/1/viga-ipe-hea-ou-heb/

**Comment by nelsonic** [comment]
Submitted a RFQ for more metal to finish the structure:

![image](https://github.com/dwyl/learn-civil-engineering/assets/194400/c0c13ce5-c12e-4a39-b2cd-75492e211888)

Specifically: going to order another 20 x `IPE 120` for the columns. 

Confirmed request received:

![peixoto-rodrigues-quote-request](https://github.com/dwyl/learn-civil-engineering/assets/194400/fed329e6-3e2a-4351-a1e2-807432f94a56)

Will follow-up with a call later today when they are open. 📞
